public/app.css

```diff
@@ -0,0 +1,6 @@
+figure {
+    max-width: 16px !important;
```

Comment:
is `!important` mandatory ?

Reply:
Yes. If we remove `!important`, custom plugin icons grow super large when you expand the sidebar.